_includes/header.html

```diff
@@ -3,6 +3,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Swift Megaminds</title>
     <link lang='css' rel="stylesheet" href="{{ site.baseurl }}/assets/css/main.css">
+    <script
```

Comment:
Minor nit picks is the linting consistent for script tag here?

Reply:
just comment if its the style being followed just to make sure and I'll approve

Reply:
I honestly don't know. I can probably condense it in one line.

Reply:
i think multi line is better keep it for better readablity just that others seemed to have kept it in one line but i like this beter